latest-schema.sql

```diff
@@ -166,6 +166,7 @@ CREATE TABLE public.developer_profile (
     id character(27) NOT NULL,
     email character varying(255) NOT NULL,
     location character varying(255) NOT NULL,
+    hourly_rate integer DEFAULT 0,
```

Comment:
last thing - can you add this as an alter statement? it will be easier to track the schema evolving and also easier when running this change on the live db